tools/ci-ts/test-helpers/common.test.ts

```diff
@@ -0,0 +1,103 @@
+import { ethers } from 'ethers'
+import { assert } from 'chai'
+import { logEvents } from './common'
+
+jest.spyOn(global.console, 'log')
+
+const mockContractName = 'TEST_CONTRACT'
+
+const mockEvents = [1, 2, 3].map(n => ({
+  topic: `topic${n}`,
+  name: `event${n}`,
+  decode: jest.fn().mockReturnValue({ [`input${n}`]: `value${n}` }),
+  inputs: [{ name: `input${n}` }],
+}))
+
+const mockEventNames = mockEvents.map(e => e.name)
+
+const mockContract = ({
```

Comment:
We could put all these mocks in their own files to keep the test file more clean